coverage-reports/generate_report.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Specify snapshot version
+SNAPSHOT_VERSION_TAG="0.8.0-SNAPSHOT"
+
+# Step 1: Change to the project directory
+echo "Changing to the project directory..."
+cd ../sdk || exit 1
+
+# Step 2: Run 'mvn clean install' in the project directory
+echo "Running 'mvn clean install' in the project directory..."
+mvn clean install
+
+# Step 3: Execute run_sc_tests.sh
+echo "Executing script for integration tests..."
+cd ../qa || exit 1
+./run_sc_tests.sh -jacoco
```

Comment:
To run these tests we need to have also BITCOIND and BITCOINCLI env var exported. Let's add a check here and if any of those are missing we can echo a warning message and skip them

Reply:
done